### ISSUE TITLE
YML-File 

### ISSUE THREAD

**Issue by rruisong** [OpenSimulationInterface/osi-validation#27]
#### Describe the bug
If we specify the highest field in yml-file without any attributes, this highest filed will not be found by the OSI-Validator. It seems that we need to have at least one attribute (no matter which one it is) in yml-file.

#### Describe how to reproduce the bug
1. Go to folder 'requirements-osi-3'
2. Open the yml file 'osi_version.yml'
3. Delete all text under 'InterfaceVersion:'
4. Save the yml file 'osi_version.yml'
5. Run 'osivalidator -r -requirements-osi-3 .\data\small_test.osi'
6. See error in screenshot.
7. Open the yml file 'osi_version.yml'
8. Add one attribute (like '  version_major:')
9. Save the yml file 'osi_version.yml'
10. Run 'osivalidator -r -requirements-osi-3 .\data\small_test.osi'
11. See it works again

#### Describe the expected behavior
Since the OSI-Validator produces always an explicit output on what is not set and just outputs only the highest field, I think I only need to define the highest field in yml file (like InterfaceVersion in osi_version.yml), if I am not interested in this field.

#### Show some screenshots

![image](https://user-images.githubusercontent.com/70255526/91280925-c3cfd400-e787-11ea-8bdb-03ca18723d19.png)

#### Describe the OS you are using
 - OS: Windows 10
 - Language: Python
 - Version 3.7

**Comment by vkresch** [comment]
Yes that's true. From the user perspective it would be more intuitive to check all the fields if they are set without defining a full skeleton in the yml files. This should be a feature to implement in the next validator release. As for now I would stick to defining the full skeleton.
